src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumesRepository.cs

```diff
@@ -81,7 +81,13 @@ public class ResumesRepository : IResumesRepository
 
         public IEnumerable<Resume> GetResumes()
         {
-            return Resumes;
+            return resumes;
+        }
+
+        public Resume PostResumes(Resume resume)
+        {
+            resumes.Add(resume);
+            return resume;
```

Comment:
in this case you should be returning just one resume not a list of resumes

Reply:
Resolved